src/manifest/parsers/graphParser.ts

```diff
@@ -160,12 +158,22 @@ export class GraphParser {
           );
         }
         case "model": {
-          const url = nodeMetaMap.get(nodeName)?.path!;
-          return new Model(nodeName, parentNodeName, url);
+          // can this ever be not there?
+          const model = nodeMetaMap.lookupByUniqueId(parentNodeName);
+          if (!model) {
```

Comment:
this changes the behaviour. in previous case it will still return model with `undefined` url. But in current behaviour returning `undefined`